src/Tools/NodeDocumentationMarkdownGenerator/DynamoDictionaryEntry.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace NodeDocumentationMarkdownGenerator
+{
+    /// <summary>
+    /// Class used to serialize an entry in the Dynamo Dictionary
+    /// </summary>
+    internal class DynamoDictionaryEntry
+    {
+        [JsonProperty("Name")]
+        public string Name { get; set; }
+
+        [JsonProperty("imageFile")]
```

Comment:
Why LowerCase first letter ?
Does this need to match with some other properties ?

Reply:
yes, this needs to match the already existing dynamo dictionary json files.